esx_service/vmdk_ops_test.py

```diff
@@ -885,22 +878,29 @@ def test_vmdkops_on_tenant_vm(self):
                                          vm_list=[self.vm1_name]) 
         self.assertEqual(None, error_info)
 
-        # run create command, which should succeed since we have DEFAULT privilege set
+        # run create command, which should fail since "default_datastore" is not set for tenant1
```

Comment:
OK. I agree with you and make the change.
Now the logic is if "default_datastore" is not specified for a tenant,  a volume will be created in the datastore where the VM lives in if the user specify the name with the short notion.